src/test/java/net/minecraftforge/debug/client/TestScreen.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) Forge Development LLC and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.debug.client;
+
+import com.mojang.blaze3d.vertex.PoseStack;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.screens.Screen;
+import net.minecraft.network.chat.Component;
+public class TestScreen extends Screen
```

Comment:
Formatting: space between last import statement and class declaration.
```suggestion
import net.minecraft.network.chat.Component;

public class TestScreen extends Screen
```